x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -332,8 +335,41 @@ export class FleetPlugin
 
     this.telemetryEventsSender.start(plugins.telemetry, core);
 
+    const logger = appContextService.getLogger();
+
+    const fleetSetupPromise = (async () => {
+      try {
+        if (this.fleetSetupStatus === 'pending' || this.fleetSetupStatus === 'complete') {
```

Comment:
`fleetSetupStatus` is now useless with the last change to IIFE right?

Reply:
Yep that is correct. Removing now.